src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -110,15 +108,39 @@ function getAvailableHeight(
 }
 
 export const QueryResultPane = () => {
-    const classes = useStyles();
     const state = useContext(QueryResultContext);
+    const [fontFamily, setFontFamily] = useState("");
+    const [fontSize, setFontSize] = useState("");
+
+    useEffect(() => {
+        const getFontFamily = async () => {
```

Comment:
what does this look like if these settings aren't set?  they should default to `var(--vscode-editor-font-size)` and `var(--vscode-editor-font-family)`

Reply:
Looks like the default behavior is the UI font, not the editor font.

Without these settings, it should look like this:
![image](https://github.com/user-attachments/assets/6c8042df-eb19-4e8a-ab91-e575227d8e25)